models/model.js

```diff
@@ -56,3 +55,17 @@ exports.fetchArticlesSortedByDate = (order) => {
     return article.rows;
   });
 };
+
+exports.fetchArticleComments = (article_id) => {
+  return db
+    .query("SELECT * FROM comments WHERE comments.article_id=$1", [article_id])
+    .then((articleComments) => {
+      if (articleComments.rows.length === 0) {
+        return Promise.reject({
+          status: 404,
+          msg: `Comments with article ID:${article_id} ID doesn't exist`,
+        });
+      }
+      return articleComments.rows;
+    });
```

Comment:
articleComment.rows.length === 0 has two possibilities 1) 404 article doesn't exist;  2)200 no comments related to this article. Think of a way to deal with both the situations